package.json

```diff
@@ -60,6 +60,7 @@
     "@emotion/react": "11.11.1",
     "@emotion/styled": "11.11.0",
     "downshift": "6.1.7",
+    "esbuild": "^0.19.10",
```

Comment:
I had some esbuild issues as well. Can you try running this without `esbuild`? If we can remove this dependency, that'd be great, otherwise it's okay to keep.

Reply:
It's running fine without `esbuild` now so I removed it.. weird though